Assets/MixedRealityToolkit/Inspectors/Setup/BuildDeployWindow.cs

```diff
@@ -0,0 +1,1464 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
> Assets/MixedRealityToolkit/Utilities/BuildAndDeploy/BuildDeployWindow.cs -> Assets/MixedRealityToolkit/Inspectors/Setup/BuildDeployWindow.cs

I'm not sure this rename makes sense. This isn't an inspector but a standalone editor window.

Reply:
Why not keep it contained to the existing `BuildAndDeploy` folder, where all/most of the associated code already lives?

Reply:
I was trying to start separating out the UI layer from the model and control layers. Looking at this again, I feel like this should go under MRTK.Tools?

Reply:
Especially since it's effectively grouped with the others

![image](https://user-images.githubusercontent.com/25975362/70943004-b19c7400-2004-11ea-8e72-726db432f7c5.png)